<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadLevelerMonitor::Widgets::FileViewerWidget</class>
 <widget class="QWidget" name="LoadLevelerMonitor::Widgets::FileViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Viewer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>File Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="file_path_line_edit"/>
     </item>
     <item>
      <widget class="QToolButton" name="refresh_tool_button">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="file_context_browser"/>
   </item>
  </layout>
  <action name="action_refresh">
   <property name="icon">
    <iconset resource="../loadleveler_monitor.qrc">
     <normaloff>:/loadleveler_monitor/images/refresh.png</normaloff>:/loadleveler_monitor/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../loadleveler_monitor.qrc"/>
 </resources>
 <connections/>
</ui>
